data-prepper-plugins/s3-sink/README.md

```diff
@@ -0,0 +1,65 @@
+# S3 Sink
+
+This is the Data Prepper S3 sink plugin that sends records to an S3 bucket via S3Client.
+
+The S3 sink plugin supports OpenSearch 2.0.0 and greater.
+
+## Usages
+
+The s3 sink should be configured as part of Data Prepper pipeline yaml file.
+
+## Configuration Options
+
+```
+pipeline:
+  ...
+  sink:
+    - s3:
+        aws:
+          region: us-east-1
+          sts_role_arn: arn:aws:iam::123456789012:role/Data-Prepper
+          sts_header_overrides:
+        max_retries: 5
+        bucket:
+          name: bucket_name
+          object_key:
+            path_prefix: my-elb/%{yyyy}/%{MM}/%{dd}/
+        threshold:
+          event_count: 2000
+          maximum_size: 50mb
+          event_collect: 15s
+        codec:
+          ndjson:
+```
+
+## Configuration
+
+- `aws_region` (Optional) : The AWS region to use for credentials. Defaults to [standard SDK behavior to determine the region](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html). Defaults to `none`.
```

Comment:
These configurations are misleading, since the parameters are under `aws`, and not `aws_region`. Please follow the documentation format from the s3 source that breaks the aws options out (https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/s3-source#aws-configuration)

Reply:
Resolved.